javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,25 @@
+ let peterView = {
+  height: 10,
+  width: 8,
+  depth: 10,
+  housePrice: 2500000,
+  gardenSizeIn: 100,
+}
+let volumeInMeters = peterView.height * peterView.width * peterView.depth;
+console.log(volumeInMeters);
+peterView.housePrice = volumeInMeters * 2.5 * 1000 + peterView.gardenSizeIn * 300;
```

Comment:
You are overwriting the price with the one you calculated instead of comparing. What should be done here is to compare the price you calculated with the one actually being paid and print a message to describe whether it is a fair price or not.